src/applications/representative-search/components/results/SearchResult.jsx

```diff
@@ -264,6 +265,7 @@ const SearchResult = ({
               text="Report outdated information"
               label={`Report outdated information for ${officer}`}
               uswds
+              disableAnalytics
```

Comment:
I think this needs to be `disable-analytics` (kabab case) because you're not using the React binding.